frappe/patches.txt

```diff
@@ -1,5 +1,5 @@
 [pre_model_sync]
-frappe.patches.v15_0.remove_implicit_primary_key
```

Comment:
Changing this won't do anything.


Move this entry at very top and leave the version number same. That way it will get executed before anything from ERPnext or Frappe runs.

Reply:
Fyi: patches run from top to bottom, pre model sync run before doctypes are migrated, post ones run after.

Reply:
It worked for you probably because it already ran once and retrying it made it work.